### DISCUSSION TITLE
What is the difference between 'sync failed' and 'degraded'

### DISCUSSION THREAD

**Discussion by dongho-jung** [argoproj/argo-cd#17409] [Q&A] [answered]
[The official documentation](https://argo-cd.readthedocs.io/en/stable/operator-manual/health/) only says that "degraded is degraded", so I don't really understand the difference between the two.

In practice, when I press sync and for some reason the health is not okay, it waits for 15 minutes and then changes to degraded.
But I don't understand how this is different from the 'sync failed' status.

**Comment by crenshaw-dev** [comment, marked as answer]
"Sync failed" generally means that something went wrong when attempting to "kubectl apply." Although, other failures may be brought into the sync operation context, such as sync hooks. So if a sync hook Job fails, the sync is considered to also be failed.

"Degraded" means that one or more of the resources monitored as part of your Application is failing its health check. The reason could be any number of things. This health state is constantly updated, unlike sync status, which is updated only during a sync operation.

**Reply by dongho-jung** [reply]
Thank you 😊

**Reply by cdsandoval** [reply]
thanks

**Reply by Velyks** [reply]
@crenshaw-dev Does ArgoCD respect the probe's `failureThreshold` when reporting something as degraded? or does it just take one event of a failing health check (readiness or liveness) to become degraded until a successful health check is made?
